.github/actions/build/action.yml

```diff
@@ -0,0 +1,56 @@
+
+name: Build
+description: Build and push docker images with Docker Buildx
+inputs:
+  DOCKERHUB_USERNAME:
+    required: true  
+    description: The username to authenticate with dockerhub
+  DOCKERHUB_TOKEN:
+    required: true 
+    description: The token to authenticate with dockerhub
+  TEMURIN_TAG:
+    required: true 
+    description: The eclipse temurin docker tag
+  IMAGE:
+    required: true
+    description: Either JDK or JRE
+
+runs:
+  using: composite
+  steps:
+    # https://github.com/docker/setup-qemu-action
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    # https://github.com/docker/setup-buildx-action
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to Docker Hub
+      if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+      uses: docker/login-action@v1
+      with:
+        username: ${{ inputs.DOCKERHUB_USERNAME }}
+        password: ${{ inputs.DOCKERHUB_TOKEN }}
+
+    # https://github.com/docker/metadata-action
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        # list of Docker images to use as base name for tags
+        images: |
+          dwolla/docker-java/${{ inputs.IMAGE }}
```

Comment:
The image name here seems inconsistent with what's being done in the Makefile. Should this be `dwolla/java`? Whether it's a JRE or JDK image will be noted as part of the `TEMURIN_TAG` input value (e.g. `8u322-b06-jre` or `11.0.14.1_1-jdk`)